h5py/_hl/base.py

```diff
@@ -11,6 +11,7 @@
     Implements operations common to all high-level objects (File, etc.).
 """
 
+import numpy as np
```

Comment:
Shouldn't the numpy import come after the standard library imports?

Reply:
Looking through the other `_hl` modules, it doesn't look like we've got a consistent way of organising imports. Do we want to impose one?

I tend to combine stdlib and third-party imports in one alphabetical list, mixing `from` imports in with regular imports, but separate local imports. But I'm not very consistent about any of that, and I'm happy to follow another convention if we like.

Reply:
I guess I always follwo pep8, where the 3rd party imports come after stdlib ones. We should probably impose at least that :wink:

Reply:
Ah, I didn't know PEP 8 made recommendations about import order. I've rearranged these ones now.